Queries.php

```diff
@@ -0,0 +1,287 @@
+            "ALTER TABLE `cr_cc_asr_mou` DROP INDEX `unique_report_date_carrier_id_traffic_type`, ADD UNIQUE `unique_report_date_carrier_id_traffic_type_roaming` (`report_date`,`carrier_id`,`traffic_type`,`roaming`)",
+            "ALTER TABLE `total_mou_on_wk_day` DROP INDEX `total_mou_on_wk_day_date_traffic_type_unique`, ADD UNIQUE `total_mou_on_wk_day_date_traffic_type_roaming_unique` (`report_date`,`traffic_type`,`roaming`)"
+        ];
+
+        foreach ($alterQueries as $query) {
+            $statement = $conn->prepare($query);
+            if (!($statement->execute())) {
+                throw new Exception('exception occured in alterQueries');
+            }
+    
+        }
+    }
+
+   
+
+    function insertIntoMsrn($conn){
+        $query = "INSERT INTO msrn_ranges (from_msisdn, to_msisdn) VALUES (94783502000, 94783502999), (94783503000, 94783503999), (94783506000, 94783506999),  (94783507000, 94783507999), (94780057000, 94780057999), (94780058000, 94780058999), (94780059000, 94780059999), (94780060000, 94780060999)";
+
+        $statement = $conn->prepare($query);
+        if (!($statement->execute())) {
+            throw new Exception('exception occured in insertIntoMsrn');
+        }
+        $statement->close();
+    }
+
+
+    function alterTableCdrCall($conn){
+        $query = "ALTER TABLE `cdr_call_test_01` ADD `roaming` TINYINT NULL DEFAULT '0' AFTER `network_id`";
+        $statement = $conn->prepare($query);
+        if (!($statement->execute())) {
+            throw new Exception('exception occured in alterTableCdrCall');
+        }
+    }
+
+    
+    function createTableSystemParameters($conn){
+        $query = "
+        CREATE TABLE `system_parameters` (
+        `id` int(11) NOT NULL,
+        `param` varchar(200) NOT NULL,
+        `value` text NOT NULL,
+        `description` varchar(256) DEFAULT NULL,
+        `created_at` datetime DEFAULT NULL,
+        `updated_at` datetime DEFAULT NULL,
+        `deleted_at` datetime DEFAULT NULL,
+        `created_by` int(11) DEFAULT NULL,
+        `updated_by` int(11) DEFAULT NULL
+        ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+        ";
+
+        $statement = $conn->prepare($query);
+        if (!($statement->execute())) {
+            throw new Exception('exception occured in createTableSystemParameters');
+        }
+    }
+
+    
+
+    function alterTableSystemParamOne($conn){
+        $query = "
+        ALTER TABLE `system_parameters`
+        ADD PRIMARY KEY (`id`),
+        ADD KEY `param` (`param`)
+        ";
+
+        $statement = $conn->prepare($query);
+        if (!($statement->execute())) {
+            throw new Exception('exception occured in alterTableSystemParamOne');
+        }
+    }
+    
+
+    function alterTableSystemParmTwo($conn){
+        $query = "ALTER TABLE `system_parameters` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT";
```

Comment:
You can add auto_increment when creating a table.

Reply:
done